ci-operator/step-registry/openshift-power-monitoring/tests/kepler/openshift-power-monitoring-tests-kepler-commands.sh

```diff
@@ -6,9 +6,9 @@ must_gather() {
     oc get pods -n openshift-operators
     oc describe deployment kepler-operator-controller-manager -n openshift-operators
     oc logs -n openshift-operators deployments/kepler-operator-controller-manager
-    oc get pods -n openshift-kepler-operator
-    oc describe daemonsets/kepler-exporter-ds -n openshift-kepler-operator
-    oc logs -n daemonsets/kepler-exporter-ds -n openshift-kepler-operator
+    # oc get pods -n openshift-kepler-operator
+    # oc describe daemonsets/kepler-exporter-ds -n openshift-kepler-operator
+    # oc logs -n daemonsets/kepler-exporter-ds -n openshift-kepler-operator
```

Comment:
Is this intentional? Could you please add a note why this is commented?

Reply:
Yes, It's not ready to merge. I am still testing some bits🙂